src/components/forms/price/PriceForm.tsx

```diff
@@ -95,6 +95,7 @@ export default function PriceForm({
                   placeholder="Convert to"
                   onChange={field.onChange}
                   namespace="CURRENCY"
+                  ignore={[(defaultValues?.fk_commodity as Commodity)?.guid || '']}
```

Comment:
unrelated: Ignore the current commodity so we avoid users adding prices for EUR.EUR and so on